web/src/scripts/webflow/methods.mjs

```diff
@@ -6,7 +6,7 @@ if (typeof process != "undefined") {
 }
 
 // eslint-disable-next-line no-undef
-if (process.env.WEBFLOW_API_TOKEN == undefined) {
+if (process.env.WEBFLOW_API_TOKEN_V2 === undefined) {
```

Comment:
[pebble] This one doesn't need to change. This is referencing the name of the env variable after it's been set.